Update the readme to include a troubleshooting section also (#2)
bangthetable/genesis: README.md

The above command will prompt for installing xcode-select, after that installs, enter the password and continue with the script.
This script will clone the genesis repo into `/tmp/genesis` and then run it for you.

## Troubleshooting

> 1. Ansible fails due to errors relating to sudo/paswword or any other error:

Reviewer: ```suggestion
1. Ansible fails due to errors relating to sudo/password or any other error:
```

Rerun the script with the following command(-K is for --ask-become-pass):
```
cd /tmp/genesis
ansible-playbook -K mac.yml
```

2. Ansible fails because some utility/software was already installed:

>   Make Sure that you don't have anything pre installed when running genesis.

Reviewer: ```suggestion
  Make sure that you don't have anything pre-installed when running genesis.
```

>   If you still have something pre installed please use brew to install it and the script will not fail.

Reviewer: ```suggestion
  If you need to install something please ensure brew was used to install it or else the script will fail.
```
